theia-ide/theia
# fix '@theia' implicit dependencies

#### What it does


Fixes #5790

- fixed the `@theia/` `no-implicit-dependencies` warnings from extensions
which used dependencies directly without declaring them in their package.json
- fixed issue in `@theia/plugin-ext` regarding incorrect import paths

#### How to test
1. build an application that uses `@theia/plugin-ext` or `@theia/plugin-ext-vscode` excluding `@theia/mini-browser` as a dependency. (ensure that `@theia/mini-browser` is not being pulled by another dependency as well.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/ak/pr_template/doc/pull-requests.md#reviewing)

Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>
